tensorflow/core/framework/resource_mgr.cc

```diff
@@ -25,6 +25,26 @@ limitations under the License.
 
 namespace tensorflow {
 
+Status setOutputResourceHandle(OpKernelContext* context, int output_index,
```

Comment:
I meant capitalize the first letter, so it's Set and not set. And maybe make this a part of the public API too?